mlflow/metrics/genai/genai_metric.py

```diff
@@ -280,7 +290,9 @@ def eval_fn(
                 )
             grading_payloads.append(
                 evaluation_context["eval_prompt"].format(
-                    input=input, output=output, grading_context_columns=arg_string
+                    input=(input if include_input else None),
```

Comment:
Now the eval context doesn't need to have an explicit `include_input` parameter.